src/Map/src/Bridge/Leaflet/composer.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "symfony/ux-map-leaflet",
+    "type": "symfony-ux-map-bridge",
+    "description": "Symfony UX Map Leaflet Renderer",
```

Comment:
```suggestion
    "description": "Symfony UX Map Leaflet Bridge",
```

not too sure 😅

Reply:
Wasn't sure too, but this makes a bit more sense IMO